### DISCUSSION TITLE
Best way to represent a ligature?

### DISCUSSION THREAD

**Discussion by jsbien** [psb1558/Junicode-font#233] [Q&A] [answered]
![snippet_RubricellaUnglerTrJSB_page1x1559y130](https://github.com/psb1558/Junicode-font/assets/8024702/4f3a3708-527a-43bb-82e1-767c1c33c275)
I guess the word means 'subscripto" and for the time being I encode it as 'ſ͡ƀſcripto". Any better ideas? it's important for me to preserve the information about the ligature.

**Comment by psb1558** [comment, marked as answer]
What you have in the image, I think, is probably b with a stylized form of a crossbar as in **ƀ** U+0180 (which I agree is the right choice here). **b** will form a ligature with a preceding **ſ** U+017F, but unfortunately **ƀ** will not. You'll need to decompose it into `b U+0335`, and then that *should* give you
<img width="223" alt="image" src="https://github.com/psb1558/Junicode-font/assets/16215767/6421bc44-37b8-435d-8316-876b948e1d3d">
Or it *would* if the ligaturing form of **b** had the same anchors as regular **b**. It doesn't in the current version of Junicode, but I have added them so that this will work in the next release (probably this weekend).

**Reply by jsbien** [reply]
Thanks!
Should it work by now? Do you really mean U+0335? It doesn't work for me.
![JunicodeTest](https://github.com/psb1558/Junicode-font/assets/8024702/77948e76-5a0f-491f-9546-d31045eab813)

**Reply by psb1558** [reply]
It's working for me. Are you using XeTeX? Can you share a minimal file?

**Reply by jsbien** [reply]
[ligature.zip](https://github.com/psb1558/Junicode-font/files/12857216/ligature.zip)

**Reply by psb1558** [reply]
The file is still looking for the beta version, which definitely won't work. It works for me when I change filenames like `JunicodeTwoBeta*.ttf` to `Junicode*.ttf` (or just "Junicode" and let fontspec find the installed version):

<img width="592" alt="image" src="https://github.com/psb1558/Junicode-font/assets/16215767/7bb1efbe-c974-4c1c-9c96-b353eb25c7ff">

**Reply by psb1558** [reply]
This was my test file:
```
\documentclass[14pt]{article}
\usepackage{fontspec}
\setmainfont{Junicode}
\begin{document}

{\addfontfeature{CharacterVariant=38:2}sb\char"0335}

\end{document}

```

**Reply by jsbien** [reply]
[JunicodeTest.pdf](https://github.com/psb1558/Junicode-font/files/12857492/JunicodeTest.pdf)
I apologize for bothering you with my stupid mistake (I was confused by the strange behaviour of git, but this is another story). 
Thank you very much!
Just curious, positioning on the DOTTED CIRCLE is correct?

**Comment by psb1558** [comment]
1. No problem. 2. Yes, the  relevant anchor is in the center of the dotted circle.
